tools/toit.cmake

```diff
@@ -120,5 +120,4 @@ macro(toit_project NAME PATH)
     DEPENDS "${TOITPKG}"
   )
   add_dependencies(download_packages "${DOWNLOAD_TARGET_NAME}")
-  add_dependencies("${DOWNLOAD_TARGET_NAME}" sync_packages)
```

Comment:
Make this dependent on a CMake flag.
You could then just `make disable-auto-sync` in your local check-out, but keep the sync for typical applications.

Reply:
See https://github.com/toitlang/toit/pull/1781